draft-ietf-tls-tls13.md

```diff
@@ -1789,6 +1789,8 @@ client offers a certificate on its first flight. This
 is consistent with the server being
 able to ask for client authentication after the handshake is
 complete. See {{post-handshake-authentication}}.
+Note that a Zero-RTT handshake can either create a new session, or
+resume a previous PSK session.
```

Comment:
This isn't quite right.  PSK and resumption are the same thing.  I think that you mean to say: 0-RTT handshakes can be used with pre-shared keys.

The concept of a session in the sense you use here is no longer relevant.